elasticsearch/manifests/kustomization.yaml

```
@@ -7,9 +7,15 @@ resources:
 patches:
   # Replace the env variables containing the hostname with namespace
   - path: fix-hosts.yaml
+  # remove empty pod annotations, as in Kustomize 5.0.x kustomize generates them as strings with value "null" instead of just null, and they can not be applied.
+  # added this issue in kustomize: https://github.com/kubernetes-sigs/kustomize/issues/5171
+  - patch: |-
+      - op: remove
+        path: /spec/template/metadata/annotations
```

Maybe worth a test so we don't accidentally delete some (maybe important/useful) annotations after an update

```suggestion
      - op: test
        path: /spec/template/metadata/annotations
        value: null
      - op: remove
        path: /spec/template/metadata/annotations
```

---

Good idea, [applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/17/commits/f31dd61b196f497b19f89ea0b5da99f7224c6c71) :)